lib/smart_proxy_openscap/openscap_plugin.rb

```diff
@@ -18,6 +18,7 @@ class Plugin < ::Proxy::Plugin
     https_rackup_path File.expand_path("http_config.ru", File.expand_path("../", __FILE__))
 
     default_settings :spooldir => '/var/spool/foreman-proxy/openscap',
-                     :openscap_send_log_file => '/var/log/foreman-proxy/openscap-send.log'
+                     :openscap_send_log_file => '/var/log/foreman-proxy/openscap-send.log',
+                     :contentdir => '/var/lib/openscap/content'
```

Comment:
not sure if this are the best directories ? afaik normally the logs are in ~foreman_proxy/logs, and later converted to symlinks in packing.. @domcleal ?